### ISSUE TITLE
Show all possible enum values as examples in swagger

### ISSUE THREAD

**Issue by niksw7** [swaggo/swag#1098]
**Is your feature request related to a problem? Please describe.**
While working with enums, I noticed that the only way to show the different possible values of enum is by adding the tag example
```go
type Amount struct {
	CurrencySymbol CURRENCY `json:"currencySymbol" example:"INR,USD,EUR"`
}

const (
	INR CURRENCY = "INR"
	USD CURRENCY = "USD"
	EUR CURRENCY = "EUR"
	PKR CURRENCY = "PKR"
)
```
Now the problem with this approach is it's not fool-proof. The actual enum values can differ over the time with the example tag.
eg : if we add PKR currency as supported enum, and miss it adding in the example tag, there would be inconsistency.

**Describe the solution you'd like**
Scan the code for all the possible values of the enum, and add them as default values in swagger example field without explicit need of example tag.

**Comment by niksw7** [comment]
@ubogdan do you think this will be a good feature to contribute on?
I can probably raise a PR, if you think it's good enough

**Comment by ubogdan** [comment]
@niksw7 the example field should be polluted with all possible values. I believe is more appropriate to fill the enums values like the implementation above:
https://github.com/swaggo/swag/blob/8ffc6c29c01a13fb01183ee91d0fcc5fc586b431/field_parser.go#L224-L239

**Comment by niksw7** [comment]
yeah, i am aware of the enums tag. But it is explicit. We need to add the examples in this field
I wanted to check why can't it be implicit whenever enums tag or examples tag is empty?

**Comment by ubogdan** [comment]
Because we have a dedicated parameter where we can describe enums.
